packages/@angular/cli/lib/config/schema.json

```diff
@@ -105,6 +105,10 @@
             "type": "string",
             "description": "The name of the TypeScript configuration file for unit tests."
           },
+          "karma": {
```

Comment:
@clydin yes could be done but wanted to keep breaking change minimal as then we would have to maintain backward compatibility too for existing projects.